handler.go

```diff
@@ -789,6 +792,50 @@ func (h *Handler) handleGetFrameViews(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handleDeleteView handles Delete /frame/view request.
+func (h *Handler) handleDeleteView(w http.ResponseWriter, r *http.Request) {
+	indexName := mux.Vars(r)["index"]
+	frameName := mux.Vars(r)["frame"]
+	viewName := mux.Vars(r)["view"]
+
+	// Retrieve frame.
+	f := h.Holder.Frame(indexName, frameName)
+	if f == nil {
+		http.Error(w, ErrFrameNotFound.Error(), http.StatusNotFound)
+		return
+	}
+
+	// Check the frame for time quantum.
+	if f.TimeQuantum() == "" {
```

Comment:
The current `(f *Frame) DeleteView` function implicitly manages deleting the standard and inverse `Views`.  So the `Field` based views are a problem for the current implementation.  I'm thinking of changing the to explicitly specify the full view name i.e. standard_2016010215 and inverse_ 2016010215.  This will enable the Field delete to use the same interface.  Plus the time window retention will manage this internally so this public API will not be needed for it.